README.md

```diff
@@ -1,7 +1,8 @@
 # Open Enterprise
-*Note: Open Enterprise is the new name of That Planning Suite. We still need to update the Github repo URL.*
 
-[![Build Status](https://img.shields.io/travis/AutarkLabs/planning-suite.svg?style=flat-square)](https://travis-ci.org/AutarkLabs/planning-suite) [![Coverage Status](https://img.shields.io/coveralls/github/AutarkLabs/planning-suite.svg?style=flat-square)](https://coveralls.io/github/AutarkLabs/planning-suite)
+_Note: Open Enterprise is the new name of That Planning Suite._
+
+[![Build Status](https://img.shields.io/travis/AutarkLabs/open-enterprise.svg?style=flat-square)](https://travis-ci.org/AutarkLabs/open-enterprise) [![Coverage Status](https://img.shields.io/coveralls/github/AutarkLabs/open-enterprise.svg?style=flat-square)](https://coveralls.io/github/AutarkLabs/open-enterprise)
```

Comment:
The coverage was showing the latest result from 25 days ago, I enabled coveralls to track this repo from now, so it should show the coverage after the first build